Code/Framework/AzQtComponents/AzQtComponents/DPEDebugViewStandalone/main.cpp

```diff
@@ -70,6 +70,16 @@ namespace DPEDebugView
             NotReflected = 0xFF
         };
 
+        AZStd::vector<AZ::Edit::EnumConstant<EnumType>> GetEnumValues() const
+        {
+            AZStd::vector<AZ::Edit::EnumConstant<EnumType>> values;
+            values.emplace_back(EnumType::Value1, "Value 1");
+            values.emplace_back(EnumType::Value2, "Value 2");
+            values.emplace_back(EnumType::ValueZ, "Value Z");
+            values.emplace_back(EnumType::NotReflected, "Not Reflected (set from EnumValues)");
+            return values;
+        }
+
```

Comment:
This should be tested with an Enum type that uses negative values under the hood.
Such as an AZ::s32.

Reply:
valid point. had not considered negatives with simple enum cases.

Reply:
Good callout, updated the standalone DPE to use a signed enum field for ad-hoc testing at least.